db/schema.rb

```diff
@@ -322,7 +322,7 @@
     t.string "trs_first_name", limit: 80
     t.string "trs_last_name", limit: 80
     t.date "date_of_birth"
-    t.string "trs_induction_status", limit: 16
+    t.string "trs_induction_status", limit: 18
```

Comment:
Can we make this larger? 50? Unless there's a good reason not to. It feels like it might bite us again potentially.

Reply:
Certainly could, that was my initial reaction. Waiting to hear what the devs think.

Reply:
I don't see any problem if we increase the length to 50